src/pages/redux/redux-thunk/index.md

```diff
@@ -3,7 +3,7 @@ title: Redux Thunk
 ---
 ## Redux Thunk
 
-Redux Thunk is middleware that allows you to return functions, rather than just actions, within Redux<sup>1</sup>. This allows for delayed actions, including working with promises.
+Redux Thunk is middleware that allows you to return functions, rather than just actions, within Redux. This allows for delayed actions, including working with promises.
```

Comment:
Hi @highskillzz 

Please add the change back that you are removing on this line. The source link is at the bottom of the page for this example.

Reply:
ohh ok